builder/bundles.go

```diff
@@ -267,8 +267,11 @@ func parseNoopInstall(installOut string) ([]packageMetadata, error) {
 		// If there is no such section, e.g. dnf fails because there's
 		// no matching package (so no "Installing:"), return nil. Real
 		// failure will happen later when doing the actual install.
-
-		return nil, fmt.Errorf("unable to resolve package")
+		if len(parts) == 1 {
+			parts = strings.Split(parts[0], "No match for argument:")
+			parts = parts[1:]
+		}
+		return nil, fmt.Errorf("unable to resolve package %s", parts)
```

Comment:
Can we format the output:
**Currently**:
```
unable to resolve package [ google-chrome-stable
  test-pkg
]
```
**Suggested**:
`unable to resolve package(s): google-chrome-stable, test-pkg`

Reply:
updated